src/Plugin.cc

```diff
@@ -1,17 +1,17 @@
 
 #include "Plugin.h"
 #include "AF_Packet.h"
-#include "iosource/Component.h"
+#include "zeek/iosource/Component.h"
 
-namespace plugin { namespace Zeek_AF_Packet { Plugin plugin; } }
+namespace af_packet::plugin::Zeek_AF_Packet { Plugin plugin; }
```

Comment:
I think the `af_packet::` prefix is not the convention here. Any thoughts?

Reply:
It's honestly unclear to me.  I took my example from the updates to the test scripts  where  they use a `btest::` prefix.  However, Tim's companion pull request to this just had `zeek::`

Reply:
Looking at other plugins I went ahead using `zeek::`.